be/tool/beck.c

```diff
@@ -459,6 +464,15 @@ static bool  dry_run = false;
 static bool  disable_directio = false;
 static bool  signaled = false;
 
+static struct m0_be_tx_bulk_cfg default_tb_cfg = (struct m0_be_tx_bulk_cfg){
```

Comment:
I can not add comment for LINE 454, 455 in original code.

Can we move "struct scanner s;" and "struct builder b;" into main()?

If possible, don't define them as global variables. We can pass them from function to functions.  Using global variables is not a good idea.

Reply:
No we can't.
Scanner and builder are very large in size so we can stack overflow issue. 
i.e. scanner structure  s_chunk and and in builder b_cache.

Reply:
I see. But we still can use malloc() to allocate them from heap.

Anyway, this is just a recommendation.

Reply:
Noted, Will take care in future